### PR TITLE
Enable the cleanup job in Prow

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
This PR is to enable the cleanup job that was disabled earlier because of the debugging purpose.
Also adding the cleanup for networks older than 4hours.

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/158#" title="Author self-approved">Rajalakshmi-Girish</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: config/jobs/periodic/housekeeping/cleanup-k8s-periodics.yaml
Comment (mkumatag) [inline config/jobs/periodic/housekeeping/cleanup-k8s-periodics.yaml:27]: Let's only purge the instances and not the network, though purge command takes before arguments but internally it does not really compare with the creation time, in fact networks does not have the creation time..

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: config/jobs/periodic/housekeeping/cleanup-k8s-periodics.yaml
Comment (Rajalakshmi-Girish) [reply config/jobs/periodic/housekeeping/cleanup-k8s-periodics.yaml]: I have removed the purge of networks and re-submitted it.

**Review by mkumatag — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/158#pullrequestreview-617015767" title="LGTM">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/test-infra/pull/158#" title="Author self-approved">Rajalakshmi-Girish</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
@Rajalakshmi-Girish: Updated the `job-config` configmap in namespace `prow` at cluster `default` using the following files:
 - key `cleanup-k8s-periodics.yaml` using file `config/jobs/periodic/housekeeping/cleanup-k8s-periodics.yaml`

<details>

In response to [this](https://github.com/ppc64le-cloud/test-infra/pull/158):

>This PR is to enable the cleanup job that was disabled earlier because of the debugging purpose.
>Also adding the cleanup for networks older than 4hours.


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
